src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -316,7 +328,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
           } else {
             new Throwable(s"Task ${backendCall.workflowDescriptor.id}:${backendCall.call.name} failed: error code $errorCode. Message: $errorMessage")
           }
-          FailedExecution(throwable)
+          FailedExecution(throwable, Option(errorCode))
```

Comment:
It's like the pumping lemma. If you can keep pumping in newlines this diff is regular